src/graphene-rect.h

```diff
@@ -175,6 +175,12 @@ void                    graphene_rect_expand            (const graphene_rect_t
 GRAPHENE_AVAILABLE_IN_1_4
 const graphene_rect_t * graphene_rect_zero              (void);
 
+GRAPHENE_AVAILABLE_IN_1_8
```

Comment:
This will need to be `GRAPHENE_AVAILABLE_IN_1_10`.

Reply:
See https://github.com/ebassi/graphene/pull/141#discussion_r263202117